scm-ui/ui-webapp/src/containers/NavigationBar.tsx

```diff
@@ -111,13 +125,26 @@ const NavigationBar: FC<Props> = ({ links }) => {
     return () => window.removeEventListener("click", close);
   }, [burgerActive]);
 
+  useEffect(() => {
```

Comment:
The whole `useEffect` hook and the `useState` hook for matchQuery should be moved into a named custom hook. If the media query would be a parameter the hook could be reusable.